release-schema.json

```
@@ -1,10 +1,24 @@
 {
   "definitions": {
+    "Award": {
+      "properties": {
+        "buyers": {
+          "title": "Buyers",
+          "description": "The organizations whose budgets will be used to purchase the goods, services or works specified in this particular award. When the multiple buyer extension is used, a buyer must be specified here **unless** the buyer for this award is the same as the overall buyer specified at the release level.",
```

Use the OCDS 1.2 definition.
```suggestion
          "description": "The organizations aiming to conclude a contract with the suppliers or to use the goods, services or works resulting from the contract, if not identical to the buyers for the contracting process.",
```